tests/common/dualtor/control_plane_utils.py

```diff
@@ -167,10 +170,35 @@ def get_mismatched_ports(self, db):
 
         return not bool(mismatch_ports)
     
+    def _get_nbr_data(self, intf_name, dest_name):
+        """Fetch neighbor data"""
+        ipaddress = self._get_ipaddr(intf_name, dest_name)
+        if ipaddress == "":
+            logger.debug("Failed to fetch {}'s {} address in config db. ".format(intf_name, dest_name))
+
+            return False
+
+        cmd = "/bin/ip neigh show " + ipaddress
+        nbr_data = self.duthost.shell(cmd)['stdout']
+
+        logger.debug("Fetched neighbor entry data for {} {}: {}".format(intf_name, dest_name, nbr_data))
+        return len(nbr_data) > 0
```

Comment:
would it be better to check if the return matches a valid neighbor entry?
something like:
```suggestion
        return nbr_data[0].startswith(ipaddress + " dev")
```

Reply:
Addresse.